game-backend/src/server.ts

```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+import WebSocket from 'ws';
+import MainController from './MainController';
+
+// initialize server
+const server = http.createServer();
+const WSServer = new WebSocket.Server({
+  server,
+});
+
+/* eslint-disable no-console */
```

Comment:
Why this manual disable?

Reply:
Sorry, i missed that one. Had an echo feature in a previous commit. Updated it.